### PR TITLE
Add Android collector to docs

### PR DESCRIPTION
**Pull request by meiqimichelle**
We've had an Android test collector for ages, but have not updated our public docs. We should fix that.

This PR:
- [x] Creates a new page to house information about our Android collector
- [x] Copy-pasta docs from the [collector readme](https://github.com/buildkite/test-collector-android/blob/main/README.md)
- [x] Deletes Android from the 'coming soon' page
- [x] Adds Android to the left nav
- [x] Adds Android collector to tiles on TA docs home
- [x] Adds 'beta' tag to Android collector

### CODE REVIEW

**Comment by meiqimichelle** [comment]
Hello, @olyism ! I guess you're on deck for reviews at the moment?

This is a v quick PR, apologies -- please have a look locally. I did this the fast and dirty way (ie, in-browser edits) and have not confirmed that everything is working properly. I have another PR coming soon (.NET) so will see about pulling down properly tomorrow, if you don't get to it first. 🙇🏻‍♀️

**Comment by olyism** [comment]
hi @meiqimichelle, all good, I'll take a look :)

**Comment by meiqimichelle** [comment]
I had a moment this morning to pull down and 👀. I think everything is working the way it should. Over to you @olyism .

File: pages/test_analytics/android_collectors.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+Privacy
+Security
+Status
+Docs
+Contact GitHub
+Pricing
+API
+Training
+Blog
+About
```
Comment (olyism) [inline pages/test_analytics/android_collectors.md:117]: Please remove footer text from GitHub 🔥 

```suggestion
```

**Review by olyism — CHANGES_REQUESTED** [review]
Hi @meiqimichelle thanks for putting this PR together. Please see requested changes: 

1. Minor suggestion to add beta pill to Swift
2. Remove footer text that was unintentionally pasted in

Thanks :)

**Comment by olyism** [comment]
> Hello, @olyism ! I guess you're on deck for reviews at the moment?

Hi @meiqimichelle, apologies for the late reply. Yes, it's mostly just me on deck for reviews 😓 

> This is a v quick PR, apologies -- please have a look locally. I did this the fast and dirty way (ie, in-browser edits) and have not confirmed that everything is working properly. I have another PR coming soon (.NET) so will see about pulling down properly tomorrow, if you don't get to it first. 🙇🏻‍♀️

We have Render preview (see https://github.com/buildkite/docs/pull/1763#issuecomment-1301651974 top of this page) to test with :)

**Comment by meiqimichelle** [comment]
Thanks @olyism ! I believe I accepted all your changes. Just waiting for the preview to build to 👀 but happy for you to merge if you 👀 and everything looks good 💚

**Comment by olyism** [comment]
Resolved conflict in 756fb76040b5bee06f9f7509de135f5e64a76114

**Review by olyism — APPROVED** [review]
Checked code and preview site. LGTM ✅  Ready for merge 🚀

**Pull request merged**